### PR TITLE
Add test suite for VFS retention and instant execution

### PR DESCRIPTION
**Pull request by lptr**

### CODE REVIEW

**Review by eskatos — APPROVED** [review]
LGTM, nice simplification of the profiler config file 👍

File: santa-tracker-base.conf [hunk 1]
```diff
@@ -1,92 +1,67 @@
...
+        "--no-build-cache",
     ]
-    warm-ups = 4
-}
 
-configurationTime = ${base} {
-    tasks = ["help"]
-    warm-ups = 10
-}
+    warm-ups = 1
```
Comment (wolfs) [inline santa-tracker-base.conf]: I would leave out setting the warm-ups in the base to `1`. I think the default is as good as well.

**Review by wolfs — APPROVED** [review]

File: santa-tracker-base.conf (hunk 1 above)
Comment (lptr) [reply santa-tracker-base.conf]: 👍

**Review by wolfs — COMMENTED** [review]
👍

**Pull request merged**